docs/developer/usage_and_tools.md

```diff
@@ -76,7 +95,7 @@ Current roles in Fed-BioMed development process are:
     * Currently, Core Developers also give final approval and merge the merge requests
     * new Core Developers are chosen by the existing Core Developers among the Contributors.
 
-In terms of mapping to accounts and roles on Gitlab server:
+In terms of mapping to accounts and roles on GitHub Fed-BioMed organization:
```

Comment:
We need the `Contributors` in two categories:
- `Contributors` that dont need to belong to the fedbiomed repository or organization, and submit pull requests from forks.
- `Team Developers` that are members of the repository with `write` access and usually work on feature branches
without forking the repository. They are regular contributors working in coordinated manner with other team developers. They are chosen by existing team developers from contributors.
Will rewrite CONTRIBUTING.md and usage_and_tools.md for that

Reply:
We need to adapt the label process to:
- introduce `needs-triage` & differentiate user/developer submitted issue
- differentiate the roles of team developers / contributors